src/ui_basic/editbox.cc

```diff
@@ -265,11 +260,6 @@ bool EditBox::handle_key(bool const down, SDL_Keysym const code) {
 			}
 			return true;
 
-		case SDLK_KP_6:
-			if (code.mod & KMOD_NUM) {
-				break;
-			}
-			FALLS_THROUGH;
 		case SDLK_RIGHT:
 			if (m_->caret < m_->text.size()) {
 				while ((m_->text[++m_->caret] & 0xc0) == 0x80) {
```

Comment:
Not sure if the prein-/decrements are confusing here.

Reply:
Haven't really touched that part of the code because it is not in scope of this PR. But you can open a separate PR for this if you want to.